INSTALL.md

```diff
@@ -78,21 +81,35 @@ $ camlp5 -v
 Camlp5 version 7.03 (ocaml 4.02.3)
 ```
 
-Under Mac OS X, you may obtain ProofGeneral from http://proofgeneral.inf.ed.ac.uk/.
-It comes with installation instructions.  Your version of emacs determines
-which version of ProofGeneral you need, roughly, so some experimentation may be
-required; you may even need the current development version if your emacs is
-recent.
-
-
 ## Preparing for the installation under Linux
 
-Under Ubuntu or Debian, you may install ocaml (and ProofGeneral) with
+uUnder Ubuntu or Debian, you may install ocaml (and ProofGeneral) with
 
 ```bash
 $ sudo apt-get install build-essential git ocaml ocaml-nox ocaml-native-compilers camlp4-extra camlp5 proofgeneral proofgeneral-doc libgtk2.0 libgtksourceview2.0 liblablgtk-extras-ocaml-dev ocaml-findlib
 ```
 
+Arch Linux and Manjaro Linux:
+
+    To prepare for compiling it yourself:
+
+      sudo pacman --sync --needed archlinux-keyring
+      sudo pacman-key --populate archlinux
+      sudo pacman --sync --needed ocaml camlp5 ocaml-findlib
```

Comment:
On my Arch system I installed `ocamlfind` using opam by typing:

```
opam install ocamlfind
```

This might be a better instruction as it applies to any system that has opam installed. Some Arch users might prefer using pacman for this, but it is most likely a good idea to use opam for everything ocaml related. 

I'm not sure if one can install `camlp5` through opam as well?

Reply:
Hmm, so there are two ways to deal with the issue of having two package managers like pacman and opam. Either you try to do everything using pacman or you try to do everything ocaml related using opam and the rest using pacman. Haskell users experience the same issue with cabal/stack vs pacman, in my experience you end up with a lot more stable system if you do everything related to a specific programming language (ocaml or Haskell) using their own package manager and do the rest using pacman. So I would advice users to only install ocaml and opam using pacman and then do the rest using opam.

Reply:
Thinking more about this I think this is fine as it is. It doesn't hurt to give instructions on how to get everything to work only using pacman and experienced users will easily figure out how to set things up using opam on their own.